lib/active_shipping/carriers/fedex.rb

```diff
@@ -475,7 +475,8 @@ def parse_rate_response(origin, destination, packages, response, options)
 
       if success
         rate_estimates = xml.root.css('> RateReplyDetails').map do |rated_shipment|
-          service_code = rated_shipment.at('ServiceType').text
+          service_code = rated_shipment.at('ServiceType').try(:text)
+          raise ActiveShipping::ResponseError, "The response did not contain service type" unless service_code.present?
```

Comment:
Ah I see what you mean now. I'll try to find it to see if there's something else broken you're right